scripts/workers/remote_submission_worker.py

```diff
@@ -563,9 +565,19 @@ def run_submission(
 
     if "result" in submission_output:
         status = "finished"
-        submission_data["result"] = json.dumps(submission_output.get("result"))
+        res = submission_output.get("result")
+        new_res = []
+        for item in res:
+            for key, value in item.items():
+                temp = {
+                    "split": key,
+                    "show_to_participant": True,
+                    "accuracies": value
+                }
+                new_res.append(temp)
+        submission_data["result"] = json.dumps(new_res)
         submission_data["metadata"] = json.dumps(
-            submission_output.get("submission_metadata")
+            submission_output.get("submission_result")
```

Comment:
@rcannood we allow hosts to add additional submission metadata if they want to. `result` and `submission_metadata` are two different files accessible to participants. Can we revert the changes on this line. Rest of the PR looks good to me.